invenio_sipstore/models.py

```diff
@@ -71,24 +65,35 @@ class SIP(db.Model, Timestamp):
     agent = db.Column(JSONType, default=lambda: dict(), nullable=False)
     """Agent information regarding given SIP."""
 
+    archivable = db.Column(
+        db.Boolean(name='ck_sipstore_archivable'),
```

Comment:
Constraint name in sqlite3:
``    CONSTRAINT ck_sipstore_sip_ck_sipstore_archivable CHECK (archivable IN (0, 1)), ``